apps/api/src/app/workflows-v2/shared/map-step-type-to-result.mapper.ts

```diff
@@ -16,12 +16,18 @@ export function computeResultSchema(stepType: StepTypeEnum, payloadSchema?: JSON
   return mapStepTypeToResult[stepType];
 }
 
-function buildDigestResult(payloadSchema?: JSONSchema) {
+function buildDigestResult(payloadSchema?: JSONSchema): JSONSchema {
   return {
     type: 'object',
     properties: {
       events: {
         type: 'array',
+        properties: {
+          // the length property is JS native property on arrays
+          length: {
+            type: 'number',
+          },
+        },
```

Comment:
This is the valid `JSONSchema` syntax. So here we define the `length` property on the `events` array. It will be available on arrays as it's JS native property on the arrays.